dashboard/app/assets/javascripts/components/abuse_error.jsx

```diff
@@ -0,0 +1,32 @@
+window.dashboard = window.dashboard || {};
+
+/**
+ * A component containing some text/links for projects that have had abuse
+ * reported. This is used in our blocking AbuseBox, in the share dialog, and
+ * in our smaller alert in apps.
+ */
+window.dashboard.AbuseError = (function (React) {
+  return React.createClass({
+    propTypes: {
+      i18n: React.PropTypes.shape({
+        tos: React.PropTypes.string.isRequired,
+        contact_us: React.PropTypes.string.isRequired
+      }).isRequired,
+      className: React.PropTypes.string,
+      style: React.PropTypes.object,
+      textStyle: React.PropTypes.object
+    },
+    render: function () {
+      return (
+        <div className={this.props.className} style={this.props.style}>
+          <p style={this.props.textStyle}
+              dangerouslySetInnerHTML={{__html: this.props.i18n.tos}}>
```

Comment:
This prop is appropriately frightening :smile:.  The [docs](https://facebook.github.io/react/tips/dangerously-set-inner-html.html) recommend not using `dangerouslySetInnerHTML` and `__html` in the same statement (so the `{__html: ''}` object is clearly intentional by the caller) but since this is just an i18n string I think it's ok here.